sdks/python/apache_beam/io/jdbc.py

```diff
@@ -355,3 +359,89 @@ def __init__(
         ),
         expansion_service or default_io_expansion_service(classpath),
     )
+
+
+@LogicalType.register_logical_type
+class JdbcDateType(LogicalType[datetime.date, MillisInstant, str]):
```

Comment:
these logical types handles legacy DATE and TIME type from Java SDK, it was not defined in the original portable schema doc i.e., not standard logical type. So put them under jdbc.py instead of typehint/schemas.py (place for standard logical type). Java side these types are also in JdbcIO module (not in java core)

Reply:
We should add that in a comment in the code